dictionary.txt

```diff
@@ -0,0 +1,155 @@
+.gitignore
+100MB
+10MB
+256kb
+2FA
+300ms
+3MB
+APIs
+AUR
+AppImage
+AppKit
+AppStore
+AppVeyor
+Auto-config
+BBEdit
+Chocolatey
+CircleCI
+CodeMirror
+Ctrl
+Dev
+Devtron
+DialogContent
+DialogError
+Dialogs
+ESLint
+F7
+GIFs
+GUID
+Github
```

Comment:
This should be `GitHub` — that’s the official name of the company.

Reply:
Agreed! There are a few other questionable entries in the dictionary, like `behaviour` and `favourite` when most of the documentation is written in American English. What do you think about removing those as well?

Reply:
Sounds good 👍

Reply:
We should probably standardize on US English, much to my chagrin :trollface: